packages/web/styles/globals.css

```diff
@@ -100,15 +100,15 @@
   }
 
   thead > tr > th:first-child {
-    @apply rounded-l-full py-[18px] pl-10 pr-3 md:rounded-none;
+    @apply rounded-tl-lg py-[18px] pl-10 pr-3 md:rounded-none;
```

Comment:
This will affect all pools in the app. We should confirm with design if this is intended.